sentinelhub/fis.py

```diff
@@ -29,10 +28,27 @@ def get_request(self, request):
         :type request: OgcRequest or GeopediaRequest
         :return: list of DownloadRequests
         """
-        return [DownloadRequest(url=self.get_url(request=request, geometry=geometry),
-                                filename=self.get_filename(request, geometry),
-                                data_type=MimeType.JSON, headers=OgcConstants.HEADERS)
-                for geometry in request.geometry_list]
+
+        return [self._create_request(request=request, geometry=geometry) for geometry in request.geometry_list]
+
+    def _create_request(self, request, geometry):
+
+        params = self._get_common_url_parameters(request)
+        post_data = {**params, **self._get_fis_parameters(request, geometry)}
+        # post data keys on SH are expected to be in lowercase:
+        post_data = {k.lower(): v for k, v in post_data.items()}
+
+        url = self.get_base_url(request)
+        authority = self.instance_id if hasattr(self, 'instance_id') else request.theme
+
+        headers = OgcConstants.HEADERS
+        headers['Content-Type'] = 'application/json'
+
+        return DownloadRequest(url=f'{url}/{authority}',
```

Comment:
You are using f-strings, that is why Travis CI fails for Python 3.5.
https://travis-ci.org/sentinel-hub/sentinelhub-py/jobs/593895803#L443

Reply:
Not the appropriate place for such discussion, but any particular reason we keep 3.5 compatibility?